support/docker/production/docker-compose.yml

```diff
@@ -47,15 +47,16 @@ services:
 
   postgres:
     image: postgres:10-alpine
-    environment:
-      POSTGRES_USER: ${PEERTUBE_DB_USERNAME}
-      POSTGRES_PASSWORD: ${PEERTUBE_DB_PASSWORD}
-      POSTGRES_DB: peertube
+    env_file:
+      - .env
     volumes:
       - ./docker-volume/db:/var/lib/postgresql/data
     restart: "always"
     labels:
       traefik.enable: "false"
+    networks:
+      default:
+        ipv4_address: 172.20.0.3
```

Comment:
Why is it necessary?

Reply:
My instance was struggling to get the internal docker DNS resolution to work. I figured since you'd taken the time to define the network's internal IP when we could do something like my definition for my Traefik proxy network.
```YAML
networks:
  default:
    ipam:
      driver: default
      config:
      - subnet:  172.20.0.0/16
  traefik:
    external: true
```
I figured why not I could remove that or you could before this merges, but the rest should probably merge so that we can have one unified location for configuration if not in the `docker-compose.yml` itself the `.env` can be used.

Reply:
Removed other change so the envvar change can be merged.